core/services/fluxmonitorv2/flux_monitor_test.go

```diff
@@ -373,12 +374,25 @@ func TestFluxMonitor_PollIfEligible(t *testing.T) {
 					UpdatedAt: big.NewInt(100),
 				}, nil)
 				tm.pipelineRunner.
-					On("ExecuteRun", context.Background(), pipelineSpec, mock.Anything, pipeline.JSONSerializable{
-						Val: map[string]interface{}{
-							"latestAnswer": float64(10),
-							"updatedAt":    float64(100),
+					On("ExecuteRun", context.Background(), pipelineSpec, pipeline.NewVarsFrom(
+						map[string]interface{}{
+							"jobRun": map[string]interface{}{
+								"meta": map[string]interface{}{
+									"latestAnswer": float64(10),
+									"updatedAt":    float64(100),
+								},
+							},
+							"jobSpec": map[string]interface{}{
+								"databaseID":    int32(0),
+								"externalJobID": uuid.UUID{},
+								"name":          "",
+							},
+							"meta": map[string]interface{}{
```

Comment:
Why is it duplicated down here

Reply:
This was because of the duplicate meta definition that we commented on above. I removed one now so this is back to what it was.